src/app/pages/create-account/create-account.page.html

```diff
@@ -5,6 +5,12 @@
 
 <ion-content>
   <form [formGroup]="signupForm" (ngSubmit)="createAccount()" novalidate>
+
+    <!-- Error messages -->
+    <span class="error ion-padding" *ngIf="formErrors?.signUp">
+        <ion-icon name="alert" color="danger"></ion-icon>There was a problem with registration.
```

Comment:
Added an error to the signup form group (rather than a form control). This way we can communicate errors back from AngularFire2/Firebase.